src/main/java/org/pesmypetcare/webservice/builders/Path.java

```diff
@@ -167,8 +168,8 @@ private static void checkProvidedIds(Collections collection, @NonNull String[] i
      * @return True if it is a root collection
      */
     private static boolean isRootCollection(Collections collection) {
-        return collection.equals(Collections.groups) || collection.equals(Collections.groupsNames) || collection
-            .equals(Collections.tags) || collection.equals(Collections.usernames) || collection
+        return collection.equals(Collections.groups) || collection.equals(Collections.groups_names) || collection
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Boolean expression complexity is 5 (max allowed is 3).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5534760)